lib/reader.js

```diff
@@ -69,7 +69,10 @@ module.exports = async (feeds, options) => {
         })
     );
 
-    const bundler = browserify({ entries, debug: opts.sourceMaps });
+    const bundler = browserify(entries, {
+        debug: opts.sourceMaps,
+        extensions: [".tsx", ".ts"]
```

Comment:
This extends to add additional extensions rather than overwrites and replaces?

Reply:
Oups. I suumed it would add to them, but from the doc it seems like it replaces:

```By default, all files with the extensions .js, .es, .es6 and .jsx are compiled. You can change this by passing an array of extensions.

NOTE: This will override the default ones so if you want to use any of them you have to add them back.```

Fixing....